text/label/sentencepiece/apply.py

```diff
@@ -0,0 +1,54 @@
+__all__ = ["ApplySPMModelToLexiconJob"]
+
+import logging
+
+from sisyphus import *
+
+from i6_core.lib.lexicon import Lexicon
+from i6_core.text.label.subword_utils import (
+    get_lm_tokens_from_lexicon,
+    word_to_subword_in_lexicon,
+)
+from i6_core import util
+
+try:
+    import sentencepiece
+except ImportError:
+    if not hasattr(gs, "WARNING_NO_SENTENCEPIECE") or gs.WARNING_NO_SENTENCEPIECE is True:
+        logging.warning(
+            "The package 'sentencepiece' is not installed in the manager python env. Please make "
+            "sure it is installed in the python environment running the Sisyphus worker. To "
+            "suppress this warning set 'WARNING_NO_SENTENCEPIECE=False' in the settings.py"
+        )
```

Comment:
Do we really need this?

Reply:
It's also in https://github.com/rwth-i6/i6_core/blob/main/text/label/sentencepiece/train.py to make sure the sentencepiece package is installed?